CarRentals/CarRentals/Controllers/CustomerCRUD.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CarRentals.Models;
+using System;
+
+namespace CarRentals.Controllers
+{
+    public class CustomerCRUD : JsonStorage<Customer>
+    {
+        public CustomerCRUD(JsonFile configuration) : base(configuration.CustomersJsonFile)
+        {
+
+        }
+        protected override void UpdateData(Customer element, Customer toUpdate)
+        {
+            toUpdate.FirstName = element.FirstName;
+            toUpdate.LastName = element.LastName;
+            toUpdate.Adress = element.Adress;
+            toUpdate.City = element.City;
+            toUpdate.Dni = element.Dni;
+            toUpdate.Phone = element.Phone;
+            toUpdate.Province = element.Province;
+            toUpdate.Id = element.Id;
+            toUpdate.ZipCode = element.ZipCode;
+            toUpdate.LastModification = DateTime.UtcNow;
+        }
```

Comment:
So then...? When the record gets saved?

Reply:
In update method in super class. Now SaveChanges is common to all child classes.